.github/ISSUE_TEMPLATE/4-release.yml

```diff
@@ -0,0 +1,124 @@
+name: 'WooCommerce: release checklist'
+description: Create a new issue to track the next WooCommerce core release progress.
+title: "WooCommerce RELEASE_VERSION: RC, final and dot-releases progress tracking"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue provides visibility on the progress of the release process of WooCommerce core RELEASE_VERSION and 
+        centralizes any conversations about it. The ultimate goal of this issue is to keep a reference of the steps, 
+        resources, work, and conversations about this release so it can be helpful for the next contributors releasing 
+        a new WooCommerce core version.
+
+        -   WooCommerce core version to release: RELEASE_VERSION ([milestone](https://github.com/woocommerce/woocommerce/milestone/MILESTONE_ID))
+        -   Release Lead: RELEASE_LEAD
+        -   Release Date: RC release on RELEASE_DATE
+        -   Release Date: final release on RELEASE_DATE
```

Comment:
Nit, just a matter of preference, feel free to ignore the comment, but I'd do it like:
```md
- RC release date: RELEASE_DATE
- Final release date: RELEASE_DATE
```